pkgs/development/web/playwright/driver.nix

```diff
@@ -191,9 +192,25 @@ let
       runHook postInstall
     '';
 
+    setupHook = writeText "setupHook.sh" ''
+      addBrowsersPath () {
+        export PLAYWRIGHT_BROWSERS_PATH="${playwright-core.passthru.browsers}"
```

Comment:
Is it possible to override this var? if user wants to run his script with different browsers path

Reply:
Sure, you can override it as usual (by setting it in the environment before running the script).

I think overlays etc. should still work too? Or do we need to use `pkgs.playwright-driver` instead of the local `playwright-core` definition to pull in customizations? There's a few references to `playwright-core` already...

Reply:
Playwright package named `core` but in nixpkgs it was named `driver` initially and we keep it for backward compatibility. Would be nice to migrate to `core` eventually but at the moment playwright PRs are waiting for months before someone review and merge them so let's use `driver`

Reply:
Sorry, I don't mean the name, but whether we're using the definition from `pkgs` so that overrides apply. I think that if we want to make packages overridable with things like overlays or `overrideAttrs`, then we have to use `finalAttrs` or import ourselves and use that to refer to our package instead of using what we defined directly.

Let me try a few things...

Reply:
Actually I don't see any way to override the browser derivation.

I see that in https://github.com/NixOS/nixpkgs/pull/344614 the browsers were wrapped in `makeOverridable`, but that was so that a user can customize and use just the browsers (by setting `PLAYWRIGHT_BROWSERS_PATH` to point to them), not for building a Node or Python package that uses a different set of browsers in its wrapped scripts.

So, it looks like doing it that way was never possible, but we are not making it worse here.

Reply:
> Is it possible to override this var? if user wants to run his script with different browsers path

Actually forgot about the case if the user sets the variable before running `nix-shell` (and doesn't use `--pure`). Changed it so we set it only if it's unset.